Entities/Common/Building/BlobPlacement.as

```diff
@@ -8,12 +8,12 @@
 #include "Requirements.as"
 #include "RunnerTextures.as"
 
-void PlaceBlob(CBlob@ this, CBlob @blob, Vec2f cursorPos, bool repairing=false, CBlob@ repairBlob=null)
+bool PlaceBlob(CBlob@ this, CBlob @blob, Vec2f cursorPos, bool repairing = false, CBlob@ repairBlob = null)
```

Comment:
(FWIW, it seems to me that `repairing` is equivalent to `repairBlob !is null` and is not necessary anymore?)